app/launchdarkly.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { Subject } from 'rxjs/Subject';
-import * as LDClient from 'ldclient-js';
+import { LDClient } from 'ldclient-js';
```

Comment:
I'm surprised this worked. Since `initialize` is part of the default export, I would have thought the import would have changed as well:

```js
import * as LaunchDarkly from 'ldclient-js'`;
// …
LaunchDarkly.initialize()
```